internal/scaffold/ansible/watches.go

```diff
@@ -15,12 +15,23 @@
 package ansible
 
 import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+
 	"github.com/operator-framework/operator-sdk/internal/scaffold"
 	"github.com/operator-framework/operator-sdk/internal/scaffold/input"
+	"github.com/operator-framework/operator-sdk/internal/util/fileutil"
+	"github.com/operator-framework/operator-sdk/pkg/ansible/watches"
+	"github.com/prometheus/common/log"
```

Comment:
Should this be logrus?
```suggestion
	log "github.com/sirupsen/logrus"
```